dist/index.html

```diff
@@ -5,13 +5,17 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>LeaderBoard</title>
+  <link rel="stylesheet" href="style.css">
```

Comment:
- [ ] Also here remove the link to the CSS file as you are already imported it into the JS file.